### PR TITLE
Forkserver_simple uses TimeoutforkserverExecutor

### PR DESCRIPTION
**Pull request by tokatoka**
#136 
the fuzzer example now uses TimeoutForkserverExecutor

### CODE REVIEW

File: fuzzers/forkserver_simple/src/main.rs [hunk 1]
```diff
@@ -96,12 +96,16 @@ pub fn main() {
     let mut fuzzer = StdFuzzer::new(scheduler, feedback, objective);
 
     // Create the executor for the forkserver
-    let mut executor = ForkserverExecutor::new(
-        "../../libafl_tests/src/forkserver_test.o".to_string(),
-        &[],
-        tuple_list!(edges_observer, time_observer),
+    let mut executor = TimeoutForkserverExecutor::new(
+        ForkserverExecutor::new(
+            "../../libafl_tests/src/forkserver_test.o".to_string(),
```
Comment (domenukk) [inline fuzzers/forkserver_simple/src/main.rs:101]: This solution still feels a bit hackish in general :) the `.o` file should never end up in the `src` folder but somewhere in some `target/release`, but fine for now, thx

File: fuzzers/forkserver_simple/src/main.rs [hunk 2]
```diff
@@ -96,12 +96,16 @@ pub fn main() {
...
+            &[],
+            tuple_list!(edges_observer, time_observer),
+        )
+        .unwrap(),
+        Duration::new(10, 0),
```
Comment (domenukk) [inline fuzzers/forkserver_simple/src/main.rs:106]: I'd do `Duration::from_millis(5000)` or something

**Comment by tokatoka** [comment]
https://github.com/AFLplusplus/LibAFL/pull/139/commits/2e0bbbd87e51f9cfdabb09497d0bbb253b0dfe7e fixes when the SIGKILL signal is sent after the process terminates
The exitcode does not have to be SIGKILL(9)

**Comment by tokatoka** [comment]
oh, I guess I have to fix another line...

**Pull request merged**